right-sidebar: Fix keyboard shortcut icon for right night mode coloring.

synicalsyntax opened the pull request:
Before:
![screenshot at aug 21 16-43-13](https://user-images.githubusercontent.com/15116870/44434673-53bb6180-a561-11e8-881c-15b3628c2d1a.png)


After:
![screenshot at aug 21 16-42-23](https://user-images.githubusercontent.com/15116870/44434667-44d4af00-a561-11e8-8839-4b166bf3373b.png)

timabbott commented:
I think this will introduce other issues, see 165636e0c45ed03aa21a27ceae8c551b85420c37

synicalsyntax commented:
@timabbott Okay, fixed; I added some notes in the commit description, please check it if you're confused about some of the changes I made. Let me know if anything else needs to be fixed, thanks!

timabbott commented:
@shubham-padia can you review?

shubham-padia commented on static/styles/right-sidebar.scss:
```diff
@@ -236,12 +236,13 @@
-#sidebar-keyboard-shortcuts #keyboard-icon {
+#keyboard-icon {
     position: fixed;
+    bottom: 8px;
```
@synicalsyntax Just a suggestion, would a comment indicating that the 8px is equal to the bottom padding of `.compose-content` be helpful? I'll leave including this to your discretion.

synicalsyntax commented:
@shubham-padia Added! @timabbott Final review please?

timabbott commented:
Merged, thanks @synicalsyntax!
